### ISSUE TITLE
Empty python file in callback_plugins results in Ansible run throwing exception.

### ISSUE THREAD

**Issue by zarlant** [ansible/ansible#14632]
##### Issue Type:
- Bug Report
##### Ansible Version:

```
ansible 2.0.0.2
```
##### Ansible Configuration:

N/A
##### Environment:

N/A
##### Summary:

Python files placed in a callback_plugins directory without a CallbackModule class causes ansible run to fail.
##### Steps To Reproduce:
1. Create callback_plugins directory.
2. Create an empty .py file in that directory.
3. Run any basic playbook using ansible-playbook -i hosts my_playbook.yml
4. See Ansible throw an exception:
##### Playbook Example:

```

---

- name: Sleep
  hosts: localhost
  tasks:
   - name: Sleep for a bit
     pause: seconds=1
```
##### Expected Results:

A warning should appear stating that an empty callback plugin was found, or was missing the CallbackModule class. But the code should still run; just not having loaded that callback plugin.
##### Actual Results:

```
Unexpected` Exception: 'module' object has no attribute 'CallbackModule'
```

**Comment by bcoca** [comment]
can you show the full stack trace? use -vvvv

**Comment by zarlant** [comment]
Here you go:

```
ERROR! Unexpected Exception: 'module' object has no attribute 'CallbackModule'
the full traceback was:

Traceback (most recent call last):
  File "/Users/thompz/.virtualenvs/congo_ansible/bin/ansible-playbook", line 86, in <module>
    sys.exit(cli.run())
  File "/Users/thompz/.virtualenvs/congo_ansible/lib/python2.7/site-packages/ansible/cli/playbook.py", line 150, in run
    results = pbex.run()
  File "/Users/thompz/.virtualenvs/congo_ansible/lib/python2.7/site-packages/ansible/executor/playbook_executor.py", line 79, in run
    self._tqm.load_callbacks()
  File "/Users/thompz/.virtualenvs/congo_ansible/lib/python2.7/site-packages/ansible/executor/task_queue_manager.py", line 156, in load_callbacks
    for callback_plugin in callback_loader.all(class_only=True):
  File "/Users/thompz/.virtualenvs/congo_ansible/lib/python2.7/site-packages/ansible/plugins/__init__.py", line 352, in all
    obj = getattr(self._module_cache[path], self.class_name)
AttributeError: 'module' object has no attribute 'CallbackModule
```

**Comment by bcoca** [comment]
can you test against 2.0.2? I cannot seem to reproduce this issue there

**Comment by zarlant** [comment]
Looks like this got fixed in 2.1.

Closing.
